app/styles/front.scss

```diff
@@ -503,54 +503,56 @@ $autre-lighter: #cce4e7;
   color: $text-color;
 }
 
-.result {
-  &.panel-opened {
-    .panel-heading {
-      background-color: whitesmoke;
-    }
-  }
+.table-result {
+  > tbody {
+    border: 1px solid #ddd;
 
-  .panel-heading {
-    background-color: white;
-    padding: 5px 0;
-
-    &:hover {
-      background-color: darken(whitesmoke, 3%);
+    > tr.result {
+      background-color: white;
+      cursor: pointer;
+      min-height: 2em;
     }
 
-    .montant {
-      font-weight: 600;
-      font-size: 18px;
+    > tr.result-needs-n-2 {
+      background-color: whitesmoke;
+      color: #888;
     }
 
-    .montant-inconnu {
-      position: relative;
-      top: .25em;
+    > tr {
+      background-color: whitesmoke;
     }
 
-    .montant-detail {
-      font-size: 12px;
+    > tr > th {
+      vertical-align: middle;
+      font-size: 16px;
+      font-weight: 400;
     }
   }
 
-  .panel-title .description {
-    @media (min-width: $screen-sm-min) {
-      padding-top: 0.5em;
-    }
+  .montant {
+    font-weight: 600;
+    font-size: 18px;
```

Comment:
I prefer relative (`%`) font sizes to increase reusability of components. If this is intentionally a fixed size, it should at least be expressed with `rem`.